### PR TITLE
Join a room

### PR DESCRIPTION
**Pull request by hazem-89**

### CODE REVIEW

**Comment by mirrenil** [comment]
Skriv gärna en beskrivning om vad som har gjorts när du gör en PR :)

**Comment by liinneea-a** [comment]
The chat messages should be placed inside the "chatsDiv" in chatRoom.tsx and the onClick on the "send" button should be changed to onSubmit on the form-tag so the enter-btn works.

File: client/src/components/ChatRoom.tsx [hunk 1]
```diff
@@ -1,95 +1,165 @@
...
+          <form style={formStyle} onSubmit={handleSubmit}>
+            <input
+              value={value}
+              onChange={handleChange}
+              style={inputStyle}
+              type="text"
+              placeholder="Join the conversation..."
+              ref={newMessageRef}
+            />
+            <button onClick={handleSendMessage}>SEND</button>
```
Comment (liinneea-a) [inline client/src/components/ChatRoom.tsx:97]: Call this function on "onSubmit" instead. You can remove the current "handleSubmit"

File: client/src/components/ChatRoom.tsx [hunk 2]
```diff
@@ -1,95 +1,165 @@
...
+  const handleOnLeave = () => {
+    leaveRoom();
+    navigate('/lobby');
+  };
+  if (!roomId) {
+    return <div />;
+  }
+  return (
+    <div style={rootstyle}>
+      <div style={chatsDivStyle}>
```
Comment (liinneea-a) [inline client/src/components/ChatRoom.tsx:67]: Place the messages inside this div instead :)

**Review by liinneea-a — CHANGES_REQUESTED** [review]

**Pull request merged**